src/js/jsx/sections/layers/LayerFace.jsx

```diff
@@ -408,10 +418,12 @@ define(function (require, exports, module) {
                 this._isDragEventTarget = false;
             }
             
-            this.setState({
-                isDragging: false,
-                dragStyle: null
-            });
+            if (this.isMounted()) {
```

Comment:
@shaoshing Please refer to https://github.com/adobe-photoshop/spaces-design/pull/3445 for ways I've accomplished this. 

At a cursory glance, I don't see the need for this check. `_getDropPosition` is called only by `_handleDragTargetMove`, and you call setState in there with no checks, which is part of the render callbacks.